app/models/concerns/rubygem_searchable.rb

```diff
@@ -12,6 +12,9 @@ class SearchDownError < StandardError; end
     delegate :index_document, to: :__elasticsearch__
     delegate :update_document, to: :__elasticsearch__
 
+    # updates index when new version is pushed
+    after_commit -> { delay.update_document }, on: :update
```

Comment:
Thanks for pointing it out. Actually we don't need it at all.

All this time I was thinking that `index_document` will create index when a new gem is created. Since, new versions go through the same process of `Pusher`, same `index_document` has us covered there.